doc/data-mining-library/source/reference/data.io.rst

```diff
@@ -9,7 +9,6 @@ Loading and saving data (``io``)
 * Comma-separated values (\*.csv) file,
 * Tab-separated values (\*.tab, \*.tsv) file,
 * Excel spreadsheet (\*.xls, \*.xlsx),
```

Comment:
Could you remove .xls here? We dropped support a while ago.

Reply:
I tried it and it got loaded normally, this is why I kept it. 😮

Reply:
io.py, L343. I am keeping it.